R/rw.R

```diff
@@ -0,0 +1,227 @@
+#' Reweighting factor container
```

Comment:
you seem to be missing the .Rd files for this so I'm guessing you miss the commit-hook which runs ```devtools::docoument()```. See compphys:1971 for details on how to set it up.

Reply:
Now I added the Rd file for the rw functions